### ISSUE TITLE
relatedoptions not correctly set since commit 8fc7a3e

### ISSUE THREAD

**Issue by dcpurton** [plk/biber#123]
Consider this MWE:

```TeX
\documentclass{article}
\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@book{a,
  author = {Name, A. N.},
  title = {A Book}
}
@book{b,
  author = {Name, Another},
  title = {A Different Book},
  related = {a},
  relatedstring = {Related Book:},
  relatedoptions = {skipbib=false}
}
\end{filecontents}
\usepackage[style=verbose]{biblatex}
\addbibresource{\jobname.bib}
\usepackage{parskip}
\pagestyle{empty}
\begin{document}
\cite{b}
\printbibliography
\end{document}
```

entry `a` does not appear in the bibliography, as expected. Other changes in `relatedoptions` are also ignored, e.g., `useauthor=false`.

The fact that things worked in my tex.stackexchange question must have been a quirk of `a` being cited directly in the first refsection.

**Comment by dcpurton** [comment]
It still has problems with custom entry options. (Although, I'm not 100% convinced that I've implemented things, right.)

Here's a MWE that includes the custom option `usevolume` (which I want to be `true` by default). In the `.bbl` file, notice that `usevolume` is set without an explicit value (leading to a value of `true`) in the options of the related clone, when it should be `false`. There is no problem if `a` is not cited directly in the example below.

biber also gives a perl warning:

```
Use of uninitialized value in lc at /home/dcpurton/development/biber/blib-dev/blib/lib/Biber/Utils.pm line 883.
```

```TeX
\documentclass{article}
\usepackage{filecontents}
\begin{filecontents}{\jobname.bib}
@book{a,
  author = {Name, A. N.},
  title = {A Book}
}
@book{b,
  author = {Name, Another},
  title = {A Different Book},
  related = {a},
  relatedstring = {Related Book:},
  relatedoptions = {usevolume=false}
}
\end{filecontents}
\usepackage[style=verbose]{biblatex}
\addbibresource{\jobname.bib}
\newtoggle{blx@usevolume}
\DeclareBibliographyOption{usevolume}[true]{%
  \settoggle{blx@usevolume}{#1}}
\DeclareTypeOption{usevolume}[true]{%
  \settoggle{blx@usevolume}{#1}}
\DeclareEntryOption{usevolume}[true]{%
  \settoggle{blx@usevolume}{#1}}
\ExecuteBibliographyOptions{usevolume}
\begin{document}
\cite{a}
\cite{b}
\printbibliography
\end{document}
```

**Comment by plk** [comment]
This was a bug, should be fixed with regression test, now in 2.6 DEV.
